students/Tim Pauley

```diff
@@ -0,0 +1 @@
+
```

Comment:
You'll want to remove this file before this gets approved and merged.  Should be able to do:
$ git rm Tim\ Pauley
Let me know if you have any issues.

Reply:
or:

git rm "Tim Pauley"

spaces are a bit of a pain in filenames...

Reply:
@timothyjpauley - I believe we resolved this during class - please push that change to update this PR.

Reply:
The issue is now resolved